nss-day-cohort-41/tabloid-fullstack-first-amendment-dancers-andy
# Ak user profile css

# Description
small css changes, turned category and user list to be a table, fixes resize on user detail image and added image to home page
Fixes # (23, 24)
## Type of change
Please delete options that are not relevant.

# Testing Instructions
``Git fetch origin ak-user-profile``
``Git checkout origin ak-user-profile``

can see new image on home page when logged in, category list now shows list in table instead of cards, same in user profiles. Also user profile details image resizes properly
# Checklist:
